MyMentorProject/src/test/java/com/Deny/MyMentorProject/Services/UserServiceTest.java

```diff
@@ -0,0 +1,43 @@
+package com.Deny.MyMentorProject.Services;
+
+import com.Deny.MyMentorProject.User.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class UserServiceTest {
+     private UserService userServiceUnderTest;
```

Comment:
just userService here is clear enough
```suggestion
     private UserService userService;
```